engine/src/main/java/org/terasology/engine/module/ModuleManagerImpl.java

```diff
@@ -126,7 +125,7 @@ public ModuleManagerImpl(Config config, List<Class<?>> classesOnClasspathsToAddT
     /**
      * Overrides modules in modules/ with those specified via -classpath in the JVM
      */
-    private void loadModulesFromClassPath() {
+    void loadModulesFromClassPath() {
```

Comment:
Intentional package scope change! Unicorn spotted in the wild 🦄

Reply:
@jdrueckert will approve opening this up for testing :nerd_face: